packages/pwa/app/components/_app/index.jsx

```diff
@@ -254,7 +289,9 @@ App.shouldGetProps = () => {
     return typeof window === 'undefined'
 }
 
-App.getProps = async ({api}) => {
+App.getProps = async ({api, res}) => {
+    const site = resolveSiteFromUrl(pathToUrl(res.locals.originalUrl))
+    const l10nConfig = site?.l10n
```

Comment:
```suggestion
    const l10nConfig = site.l10n
```

Looks like `resolveSiteFromUrl` would never return undefined value.. at least it'll return the default site. It's a good thing because on line 295 below, `getLocaleConfig` is expecting to see `l10nConfig` so it can know what app-supported locales are.